wordpress/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM wordpress:6.1.1
+
+LABEL maintainer="Max-test1"
+
+WORKDIR /var/www/html
+#ADD all dependence files and folders ( plugins and cors enabled )
+COPY wordpress_Dockerfile_folder/functions.php /var/www/html/wp-includes/
```

Comment:
wordpress_Dockerfile_folder - bad name
for plugins you can create folder called just plugins, for example "root/wordpress/plugins"
for includes - the same
etc.